source/fuzz/fuzzer_context.cpp

```diff
@@ -126,6 +126,7 @@ const std::pair<uint32_t, uint32_t> kChanceOfReplacingParametersWithGlobals = {
 const std::pair<uint32_t, uint32_t> kChanceOfReplacingParametersWithStruct = {
     20, 40};
 const std::pair<uint32_t, uint32_t> kChanceOfSplittingBlock = {40, 95};
+const std::pair<uint32_t, uint32_t> kChanceOfSplittingLoop = {2, 5};
```

Comment:
I found that when running fuzzer pass that even slightly higher probabilities lead to an exponential growth of number of loops. For a given loop this transformation duplicates it. Now there are two loops in which subsequent transformations SplitLoop can be applied. If probabilities are too high, the loops are continuously duplicated and I think we want to avoid that. I found that this is a maximum probability that works.